selfdrive/locationd/paramsd.py

```diff
@@ -180,6 +193,9 @@ def main(sm=None, pm=None):
       roll_std = float(P[States.ROAD_ROLL])
       # Account for the opposite signs of the yaw rates
       sensors_valid = bool(abs(learner.speed * (x[States.YAW_RATE] + learner.yaw_rate)) < LATERAL_ACC_SENSOR_THRESHOLD)
+      avg_offset_valid = check_valid_with_hysteresis(avg_offset_valid, angle_offset_average)
+      total_offset_valid = check_valid_with_hysteresis(total_offset_valid, angle_offset)
+      roll_valid = check_valid_with_hysteresis(roll_valid, roll)
```

Comment:
is this using the wrong max now?

Reply:
Fixed. Roll is in radians, offsets in degrees :(